en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: default
+title: ZigBeeAdapterTutorial
+permalink: /en-US/win10/samples/ZigBeeAdapterTutorial.htm
+lang: en-US
+---
+
+# ZigBee sample
+Get the [code](https://github.com/ms-iot/samples/blob/develop/AllJoyn/AllJoynZigBeeAdapter/ZigBeeAdapter.zip?raw=true) on Github
+
+This document describes the setup of the ZigBee adapter for Device System Bridge (DSB) on Windows 10. When using it you will be able to expose ZigBee devices to AllJoyn.
+
+## What is ZigBee?
+
+ZigBee is a low cost, low power, wireless communications standard designed to allow devices to communicate with another. In addition to the communication protocol, ZigBee standard also defines profiles, such as ZigBee Light Link or Home Automation, which themselves define device. For example, ZigBee Light Link will define what a light is, what a dimmable light is, and so on. Each device uses clusters defined in the ZigBee Cluster Library (ZCL) to specify what they can do, what they expose…
+
+See [ZigBee standard](http://www.zigbee.org) for more information about ZigBee, ZigBee profiles, ZCL clusters...
+
+Acronyms:
+- ZDO: ZigBee Device Object
+- ZCL: ZigBee Cluster Library
+
+## Prerequisites
+1. XBee ZigBee module from [Digi](http://www.digi.com), e.g.: XB24 Z7PIT-004
+2. XBee Explorer USB dongle from [SparkFun](https://www.sparkfun.com)
```

Comment:
fixed (the initial idea was to just give a link to sparkfun so if sparkfun change its web site in the future the link will remain valid)